packages/playground/vue/PreProcessors.vue

```diff
@@ -32,3 +35,10 @@ $color: magenta;
   color: @color;
 }
 </style>
+
+<style lang="stylus">
+color = orange
+
+.pug-stylus
+  color: color
+</style>
```

Comment:
We want to end files with and eol line break

```suggestion
</style>

```